frontend/tests/utils/helpers.ts

```diff
@@ -420,3 +420,15 @@ export async function takeSnapshotOfMetaTags(name: string, page: Page) {
     { page }
   );
 }
+
+export async function expandTissue(page: Page, tissueName: string) {
+  await tryUntil(
+    async () => {
+      await page
+        .getByTestId(`cell-type-labels-${tissueName}`)
+        .getByTestId("tissue-name")
+        .click();
```

Comment:
Great helper function!

we should probably add assertion here to compare the before cellType count and after cellType count has changed, otherwise Playwright is just clicking the tissue name and not checking if the tissue is truly expanded

Reply:
Should helper functions that just carry out proposed functionality do test assertions? We already have coverage on whether the expansion is working.

Reply:
Yeah I think it's probably more valuable to just do the assertion in this helper function and remove the duplicate expansion assertions at the callsites. Otherwise this helper function is not doing anything, since .click() already waits for the element to be clickable and retries until the click action registers